gdal/frmts/gtiff/geotiff.cpp

```diff
@@ -17783,13 +17811,11 @@ GTiffDataset::CreateCopy( const char * pszFilename, GDALDataset *poSrcDS,
     poDS->eAccess = GA_Update;
     poDS->m_pszFilename = CPLStrdup(pszFilename);
     poDS->m_fpL = l_fpL;
-    poDS->m_bCloseFile = true;
```

Comment:
@rouault this PR needed to be updated for 70bb4eb . I couldn't easily see any issue, could you check please?